.changeset/wicked-parents-grow.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-search-backend-module-techdocs': minor
+'@backstage/plugin-search-backend-module-explore': minor
+'@backstage/plugin-search-backend': minor
```

Comment:
Since search is 1.0 this would need to be a `major` bump.

Reply:
Ah, good call, updated. I also added `**BREAKING**` as well.